### PR TITLE
task/SM-6992: Sprint 27 sandbox release docs updates

### PR DESCRIPTION
**Pull request by ccurran01**

### CODE REVIEW

File: docs/api-documentation/index.html [hunk 1]
```diff
@@ -75,13 +75,13 @@ <h1 class="govuk-heading-m">Releases</h1>
     <li>
       <!-- UPDATE ME to upcoming latest version -->
       <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.0/">
-        Version 3.0 - Due for release to Sandbox on 29/04/2021. Due for release to Production on 04/05/2021.
+        Version 3.0 - Released to Sandbox on 29/04/2021. Due for release to Production on 04/05/2021.
```
Comment (pwright08) [inline docs/api-documentation/index.html:78]: Don't we normally have this in the current version section above, i.e. the current is updated to say "V3 - released to sandbox on X, due for release to prod on Y", then this upcoming version section would say when the next version is coming. E.g. https://github.com/departmentfortransport/street-manager-docs/pull/724/files

Comment (ccurran01) [reply docs/api-documentation/index.html:78]: 🙈

File: docs/api-documentation/V2.10.12/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: API specification V2.10.11
```
Comment (cpaul92) [inline docs/api-documentation/V2.10.12/index.md:3]:
```suggestion
title: API specification V2.10.12
```

File: docs/api-documentation/V2.10.12/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+version: 2.10.11
```
Comment (cpaul92) [inline docs/api-documentation/V2.10.12/index.md:4]:
```suggestion
version: 2.10.12
```

File: docs/api-documentation/index.html [hunk 4]
```diff
@@ -59,8 +59,15 @@ <h1 class="govuk-heading-m">Releases</h1>
 <ul class="govuk-list govuk-list--bullet">
   <ul class="govuk-list govuk-list--bullet">
     <li>
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.10/">
-        Version 2.10.10 - Released to Sandbox on 15/04/2021. Released to Production on 21/04/2021.
+      <!-- UPDATE ME to upcoming latest version -->
```
Comment (cpaul92) [inline docs/api-documentation/index.html:62]:
```suggestion
      <!-- UPDATE ME to current latest version -->
```

File: docs/api-documentation/index.html [hunk 5]
```diff
@@ -59,8 +59,15 @@ <h1 class="govuk-heading-m">Releases</h1>
...
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.0/">
+        Version 3.0 - Released to Sandbox on 29/04/2021. Due for release to Production on 04/05/2021.
+      </a>
+    </li>
+    <li>
+      <!-- UPDATE ME to upcoming stable version -->
```
Comment (cpaul92) [inline docs/api-documentation/index.html:68]:
```suggestion
      <!-- UPDATE ME to current stable version -->
```

**Review by cpaul92 — APPROVED** [review]

**Pull request merged**